docs/zello-sdk-documentation/com/zello/sdk/Zello.html

```diff
@@ -228,44 +228,56 @@ <h2 title="Class Zello" class="title">Class Zello</h2>
 </td>
 </tr>
 <tr id="i15" class="rowColor">
+<td class="colFirst"><code>boolean</code></td>
+<td class="colLast"><code><span class="memberNameLink"><a href="../../../com/zello/sdk/Zello.html#isLastMessageReplayAvailable--">isLastMessageReplayAvailable</a></span>()</code>
+<div class="block">Determines if there is a message available to replay.</div>
+</td>
+</tr>
+<tr id="i16" class="altColor">
 <td class="colFirst"><code>void</code></td>
 <td class="colLast"><code><span class="memberNameLink"><a href="../../../com/zello/sdk/Zello.html#leavePowerSavingMode--">leavePowerSavingMode</a></span>()</code>
 <div class="block">Resumes full communication between the ZelloWork app and the server.</div>
 </td>
 </tr>
-<tr id="i16" class="altColor">
+<tr id="i17" class="rowColor">
 <td class="colFirst"><code>void</code></td>
 <td class="colLast"><code><span class="memberNameLink"><a href="../../../com/zello/sdk/Zello.html#lock-java.lang.String-java.lang.String-">lock</a></span>(java.lang.String&nbsp;applicationName,
     java.lang.String&nbsp;packageName)</code>
 <div class="block">Locks the default ZelloWork app UI to prevent users from accessing it.</div>
 </td>
 </tr>
-<tr id="i17" class="rowColor">
+<tr id="i18" class="altColor">
 <td class="colFirst"><code>void</code></td>
 <td class="colLast"><code><span class="memberNameLink"><a href="../../../com/zello/sdk/Zello.html#muteContact-com.zello.sdk.Contact-boolean-">muteContact</a></span>(<a href="../../../com/zello/sdk/Contact.html" title="class in com.zello.sdk">Contact</a>&nbsp;contact,
            boolean&nbsp;mute)</code>
 <div class="block">Mutes or unmutes a contact.</div>
 </td>
 </tr>
-<tr id="i18" class="altColor">
+<tr id="i19" class="rowColor">
 <td class="colFirst"><code>void</code></td>
 <td class="colLast"><code><span class="memberNameLink"><a href="../../../com/zello/sdk/Zello.html#openMainScreen--">openMainScreen</a></span>()</code>
 <div class="block">Opens the main screen of the ZelloWork app.</div>
 </td>
 </tr>
-<tr id="i19" class="rowColor">
+<tr id="i20" class="altColor">
+<td class="colFirst"><code>void</code></td>
+<td class="colLast"><code><span class="memberNameLink"><a href="../../../com/zello/sdk/Zello.html#replayLastIncomingMessage--">replayLastIncomingMessage</a></span>()</code>
+<div class="block">Replays the last received live voice message.</div>
```

Comment:
More accurate definition of the message that will replay is *the last played message* -- it may be not the last received.